cpp/unreal_plugins/SpCore/Source/SpCore/Log.cpp

```diff
@@ -5,7 +5,6 @@
 #include "SpCore/Log.h"
 
 #include <filesystem>
-#include <format>
```

Comment:
I think the issues with `<format>` were linker errors, so it is still safe to include it here.